jquery.slimscroll.js

```diff
@@ -81,6 +81,9 @@
         // check if mousewheel should scroll the window if we reach top/bottom
         allowPageScroll : false,
 
+        // check if force vertical movement of mousewheel to scroll a area whose axis is 'x'
+        applyVerticalWheelToHolizontal : false,
```

Comment:
Awesome PR! Please could you fix this typo/naming -> `applyVerticalWheelToHolizontal` should be `applyVerticalWheelToHorizontal`